index.js

```diff
@@ -4,8 +4,9 @@ import { createLogger, format, transports } from 'winston'
 
 const API_ENDPOINT = process.env.API_ENDPOINT
 const LOG_LEVEL = process.env.LOG_LEVEL
-const TIMEOUT_SECONDS = process.env.TIMEOUT_SECONDS
 const WAIT_SECONDS = process.env.WAIT_SECONDS
+const QUEUE_TIMEOUT_WARNING = process.env.QUEUE_TIMEOUT_WARNING
+const QUEUE_TIMEOUT_MAX = process.env.QUEUE_TIMEOUT_MAX
```

Comment:
This could be
```javascript
// "destructuring" is a bit more concise of a way to 
// pull same-name variables out of things
const { 
  API_ENDPOINT,
  LOG_LEVEL, 
  WAIT_SECONDS,
  QUEUE_TIMEOUT_WARNING,
  QUEUE_TIMEOUT_MAX } = process.env
```

Reply:
Agreed, updated.